core/src/test/scala/org/apache/spark/sql/types/pk/MultiColumnPKDataTypeSuites.scala

```diff
@@ -57,11 +57,11 @@ trait MultiColumnPKDataTypeSuites extends BaseRandomDataTypeTest {
 
   protected val tests: Map[Int, Seq[(Int, Int)]] = {
     val size = dataTypes.size - 1
-    dataTypes.indices
+    val ans = dataTypes.indices
       .flatten { i =>
         dataTypes.indices
           .filter { j =>
-            i != j
+            i < j
```

Comment:
Plz make a description why?

Reply:
I think the order of the two cols in pk doesn't affect the correct. It's an enhancement that reduces the IT time and does not affect the IT results.
I will add an annotation in the code if you approve this

Reply:
You can add this in PR description

Reply:
I don't thinks so, the order of PK will affect the logic plan.